be/src/olap/push_handler.cpp

```diff
@@ -401,46 +394,42 @@ OLAPStatus PushHandler::_convert(
             // Convert from raw to delta
             VLOG(3) << "start to convert row file to delta.";
             while (!reader->eof()) {
-                if (OLAP_SUCCESS != (res = writer->attached_by(&row))) {
-                    OLAP_LOG_WARNING(
-                            "fail to attach row to writer. [res=%d tablet='%s' read_rows=%u]",
-                            res, curr_tablet->full_name().c_str(), num_rows);
-                    break;
-                }
-
-                res = reader->next(&row, writer->mem_pool());
+                res = reader->next(&row, rowset_builder->mem_pool());
                 if (OLAP_SUCCESS != res) {
-                    OLAP_LOG_WARNING("read next row failed. [res=%d read_rows=%u]",
-                                     res, num_rows);
+                    LOG(WARNING) << "read next row failed."
+                                 << " res=" << res
+                                 << " read_rows=" << num_rows;
                     break;
                 } else {
-                    writer->next(row);
+                    if (OLAP_SUCCESS != (res = rowset_builder->add_row(&row))) {
```

Comment:
AlphaRowsetBuilder add_row encapsulates attached_by function.
attached by function should be called previous by reader->next().

Reply:
OK, I will fix it